R_demos/3-wgseq-introduction.Rmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "Genomic Variants: Introduction to whole genome sequence data"
+author: Jineta Banerjee
+date: '`r format(Sys.time(), "%d %B, %Y")`'
+output: 
+  html_document:
+    code_folding: hide
+    toc: true
+    number_sections: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    fig_width: 7
+    fig_height: 6
+    fig_caption: true
+    df_print: paged 
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Getting Started
+
+This document aims to guide the hackathon participants, who have limited exposure to Synapse as a platform and clinical/biological data, through exploration and analysis of the available [whole genome sequence](https://ghr.nlm.nih.gov/primer/testing/sequencing) dataset. The available dataset is a harmonized dataset including genomic variant (WGS) data from normal and two different tumor types from two different study initiatives. 
+
+The scope of this document is to inform participants:
+
+* how to navigate the Synapse environment to access and work with the data files stored in Synapse
+* display the key features of the genomic variant (WGS) dataset 
+* introduce few general analysis steps of the data and their interpretation
+
+At the end of this document, the participants will be able to download the data and understand various intricacies that exist in this given dataset. We encourage participants to visualize the dataset by faceting on different variables to get an accurate understanding of the dataset. 
+
+```{r lib_synapser, eval=TRUE, results='hide', message=FALSE, warning=FALSE, include=TRUE}
+
+library(synapser)
+library(synapserutils)
+library(BiocManager)
+library(gProfileR)
+library(tidyverse)
+library(DT)
+library(maftools)
+library(here)
+library(ggfortify)
+library(stringr)
+library(reshape2)
+library(RColorBrewer)
+
+## The login credentials have been stored in .synapseConfig file in the home dir. If you dont have this setup, please make the file before trying to login.
+synLogin()
+
+```
+&nbsp;
+
+## Accessing or downloading data from Synapse
+
+Please look up the SynID of the relevant datafile and access it using the __synGet__ command of *synapser*. Load the required csv file containing the whole genome sequencing data directly into your Rstudio IDE by adding __$path__ to the accessed file-entity. 
+
+```{r load data, eval=TRUE, results='show', message=FALSE, warning=FALSE}
+
+#here() # check working directory
```

Comment:
suggest deleting this line, wd should be irrelevant here, right?

Reply:
Yeah..makes sense. Removed the line in the latest PR